logger/logger.go

```diff
@@ -163,15 +160,17 @@ func SimpleSetup() error {
 	return err
 }
 
-var procCmdline = "/proc/cmdline"
+// used to force testing of the kernel command line parsing
+var procCmdlineMustMockInTests = true
```

Comment:
the name vs use of this sounds kind of reversed to me

Reply:
we have a simialrly named mockMountInfoInTests, but it cases the code to panic in tests when mountinfo isn't mocked. Perhaps `procCmdlineUseDefaultMockInTests` is more fitting here then/

Reply:
procCmdlineAssumeDefaultInTests